packages/web/src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -149,43 +161,27 @@ const RewardPanel = ({
   let progressLabelFilled: string
   if (shouldShowCompleted) {
     progressLabelFilled = messages.completeLabel
-  } else if (challenge && challenge?.cooldown_days > 0) {
-    if (needsDisbursement) {
-      progressLabelFilled = messages.readyToClaim
-    } else if (pending) {
-      progressLabelFilled = messages.pendingRewards
-    } else if (challenge?.challenge_type === 'aggregate') {
-      // Count down
-      progressLabelFilled = fillString(
-        remainingLabel ?? '',
-        formatNumberCommas(
-          (challenge?.max_steps - challenge?.current_step_count)?.toString() ??
-            ''
-        ),
-        formatNumberCommas(challenge?.max_steps?.toString() ?? '')
-      )
-    } else {
-      progressLabelFilled = progressLabel ?? ''
-    }
+  } else if (needsDisbursement) {
+    progressLabelFilled = messages.readyToClaim
+  } else if (pending) {
+    progressLabelFilled = messages.pendingRewards
   } else if (challenge?.challenge_type === 'aggregate') {
     // Count down
-    progressLabelFilled = fillString(
+    progressLabelFilled = formatProgressLabel(
```

Comment:
this should be able to use the current fillString function. maybe we clean up the passed in vars though?